crates/uv-trampoline/src/bounce.rs

```diff
@@ -129,6 +145,30 @@ fn push_quoted_path(path: &Path, command: &mut Vec<u8>) {
     command.extend(br#"""#);
 }
 
+/// Checks if the given executable path is part of a virtual environment according to the procedure
+/// described in PEP 405.
+///
+/// Checks if a `pyvenv.cfg` file exists in the given executable path or that of its parent.
+/// If this file exists, checks for a `home` key.
+fn is_virtualenv(exec_dir: &Path) -> bool {
+    let Some(parent_dir) = exec_dir.parent() else {
+        return false;
+    };
+
+    for dir in &[parent_dir, exec_dir] {
+        let pyvenv_path = dir.join("pyvenv.cfg");
+        if let Ok(file) = File::open(&pyvenv_path) {
+            let reader = BufReader::new(file);
+            for line in reader.lines().map_while(Result::ok) {
+                if line.starts_with("home") {
+                    return true;
+                }
+            }
+        }
```

Comment:
Why do we need to search for this key? The presence of a `pyvenv.cfg` file alone seems sufficient?

Reply:
(We have other checks for virtual environments and they do not read the file)